core/src/main/java/com/predic8/membrane/core/interceptor/opentelemetry/OpenTelemetryInterceptor.java

```diff
@@ -127,10 +132,14 @@ private static void setSpanHttpHeaderAttributes(Header header, Span span) {
     }
 
     private void setSpanOpenAPIAttributes(Exchange exc, Span span) {
-        OpenAPIRecord record = exc.getProperty(OPENAPI_RECORD, OpenAPIRecord.class);
-        if (record != null) {
-            span.setAttribute("openapi.title", record.getApi().getInfo().getTitle());
+        OpenAPIRecord record;
+        try {
+            record = exc.getProperty(OPENAPI_RECORD, OpenAPIRecord.class);
+        } catch (ClassCastException e) {
+            log.info("No OpenAPI to report to OpenTelemetry.");
```

Comment:
can this be changed to DEBUG? (Elsewise the Log files in production will fill up very quickly.)